speaking-portal-project/tooling/frameGeneratorScript.jsx

```diff
@@ -0,0 +1,82 @@
+/*
+For internal use only - just programmatically toggles layers and exports images
+When new layers are added, these layers must be updated.
+
+Please note Photoshops scripting language is based on an ancient version of javascript, so there's no handy array
+functions
+
+This script can be run by going to Photoshop -> File -> Scripts -> Browse & selecting this script. Because it is built
+for my system, you will also have to update the file path.
+ */
+const phonemes = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'X']
+const eyes = ['blink', 'open']
+const arms = ['default', 'relaxed', 'pos1', 'pos2']
+const legs = ['legsDefault', 'legsRelaxed', 'legsPos1', 'legsPos2']
+const toggleableLayer = [
+    'A',
+    'B',
+    'C',
+    'D',
+    'E',
+    'F',
+    'G',
+    'H',
+    'X',
+    'blink',
+    'open',
+    'default',
+    'relaxed',
+    'pos1',
+    'pos2',
+    'legsDefault',
+    'legsRelaxed',
+    'legsPos1',
+    'legsPos2',
```

Comment:
Ideally poses and legs match and we can just update poses rather than individually arms & legs, but its nice to have the ability to control both too :)

Reply:
I kept it open just in case the poses the artist gives us can be mixed and matched!

Reply:
yeah makes a lot of sense! Nice :)